src/Components/addBookForm.js

```diff
@@ -1,25 +1,53 @@
-import React, { useRef } from 'react';
-import { useDispatch } from 'react-redux/es/exports';
-import uniqid from 'uniqid';
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
 import styles from './Styles/addBookForm.module.css';
-import { addBook } from '../Redux/books/books';
+// import { useDispatch } from 'react-redux/es/hooks/useDispatch';
```

Comment:
- [ ]  please try to remove the unused commented code to keep the file clean and pro

Reply:
Thanks for the feedback, 
working on changes now 👍 .